packages/design-system/CONTRIBUTING.md

```diff
@@ -1,53 +1,140 @@
 # Contributing
+Congratulations on becoming a contributor to the design system! You are pushing the whole company upwards.
 
-Please use this template in order to add a new Design System element documentation.
+Here's what we expect our contributors to do when it comes to code.
 
-Each new element should have this hierarchy
+We don't believe in giving you rigid steps to follow, we prefer guidance and a loose checklist. Always put people before processes.
 
+## How do I know what I can do?
+We have a [self-service Jira board set-up for this](https://jira.talendforge.org/secure/RapidBoard.jspa?projectKey=TUX&rapidView=1030). All the tasks in the TODO column are up for grabs!
+
+Assign yourself to a ticket, it's all fair game.
+
+You can also simply ask on Slack, channel #ask-designsystem.
+
+## TL;WR
+- Contribute to the design phase by offering browser-based know-how.
+- You may need to write the initial Storybook PR for contributing designers.
```

Comment:
is it possible to add link to the Storybook repo or to a specific guideline to contribute on it?

Reply:
This is the storybook repo :) These are the guidelines to contribute on it.

Reply:
Updated!